### PR TITLE
FIX NumbaPerformanceWarning import

### PR DESCRIPTION
**Pull request by TomDLT**

### CODE REVIEW

**Comment by agramfort** [comment]
Shouldn’t we fix the code so there are no warning ?

**Comment by mathurinm** [comment]
It is surprising that a warning is raised since we pass X as fortran, so `X[:, j]` is contiguous

**Comment by agramfort** [comment]
can you replicate @mathurinm ?

**Comment by tomMoral** [comment]
Don't we need to do the same in : benchmarks/lasso/solvers/cd.py

**Comment by tomMoral** [comment]
Maybe an helper for this would make sense? Something like `benchopt/helpers/helpers_numba.py` with a `filter_performance_warning` context?

**Comment by TomDLT** [comment]
I fixed the warnings with ordering the input array `X` using Fortran order.
It is consistently faster with the CD solver, which computes the gradient more quickly if the column are contiguous.

Figure_1
Figure_2
Figure_1
Figure_2
Figure_3

**Comment by josephsalmon** [comment]
good catch @TomDLT !

File: benchmarks/lasso/solvers/cd.py [hunk 1]
```diff
@@ -31,7 +29,8 @@ class Solver(BaseSolver):
     requirements = ['numba', 'scipy']
 
     def set_objective(self, X, y, lmbd):
-        self.X, self.y, self.lmbd = X, y, lmbd
+        # use Fortran order to compute gradient on contiguous columns
+        self.X, self.y, self.lmbd = np.ascontiguousarray(X.T).T, y, lmbd
```
Comment (agramfort) [inline benchmarks/lasso/solvers/cd.py:33]:
```suggestion
        self.X, self.y, self.lmbd = np.asfortranarray(X), y, lmbd
```

same remark for below

**Comment by agramfort** [comment]
thx @TomDLT !

**Pull request merged**